package.json

```diff
@@ -63,6 +63,7 @@
   },
   "dependencies": {
     "lodash": "^4.17.5",
-    "matrixmath": "^2.2.2"
+    "matrixmath": "^2.2.2",
+    "three": "^0.103.0"
```

Comment:
Sorry for the late answer !
We don't want to depend on a specific engine like Three, especially as we usually work with Babylonjs. I saw that three was used only in `trs-matrix.js` to break down matrices. Is it possible to implement functions like multiply, or quaternions in the lib?

Reply:
I see three options:
- writing a small implementation of affine transformation matrices, but that could be wasted time given that many tested implementations already exist.
- using a lightweight library that provides affine transformations but does not interfere with babylon, three and others.
- providing an option to the user to pass his preferred implementation of affine matrices, given a specification of what API is expected; this would most likely require the user to build an adapter class for the library he is using, since they are all different.

Personally I would prefer the second option, but I haven't been able to find a library like that.

Reply:
I know https://github.com/toji/gl-matrix develop by Tojiro (but I nevers used it), well know and tested library, feel free to take a look !

Reply:
I saw you were using it and it seemed ok, but the API looked a bit weird to me.
I'll try converting to using that when I have some time.